### ISSUE TITLE
TASK [geerlingguy.glusterfs : Import GlusterFS GPG key.] failed

### ISSUE THREAD

**Issue by weeix** [geerlingguy/ansible-role-glusterfs#11]
```
fatal: [host1.local]: FAILED! => {"changed": false, "failed": true, "msg": "failed to fetch key at http://download.gluster.org/pub/gluster/glusterfs/LATEST/rsa.pub , error was: HTTP Error 404: Not Found"}
        to retry, use: --limit @1_prepare_gluster.retry
```

Maybe the repo isn't there anymore?

**Comment by geerlingguy** [comment]
Yep; see for example, https://travis-ci.org/geerlingguy/ansible-role-glusterfs/builds/145071722

https://github.com/geerlingguy/ansible-role-glusterfs/pull/9 may fix this.

**Comment by geerlingguy** [comment]
Sort-of-fixed in #12. Moving further discussion/work into https://github.com/geerlingguy/ansible-role-glusterfs/issues/13
